src/app/App.tsx

```diff
@@ -82,7 +83,19 @@ export const App = (): JSX.Element => {
   if (authLoading || connecting || authenticating || configLoading) {
     content = <MainContentSection header={<SectionHeader loading />} />;
   } else if (!authenticated && !connectionError) {
-    content = <Login />;
+    if (authError === "Session expired") {
+      content = (
+        <MainContentSection>
+          <Notification severity="information">
+            Your session has expired. Please log in again to continue using
+            MAAS.
+          </Notification>
+          <Login />
+        </MainContentSection>
+      );
+    } else {
+      content = <Login />;
+    }
```

Comment:
Can do, yeah.

As for the `MainContentSection`, I was just wrapping it here so that the notification displays in a way that matches Amy's designs, the original <Login /> wasn't warpped so I was just keeping it consistent witht hat, but I can change it

Reply:
Yes please, thanks!